### PR TITLE
python3Packages.pillow: 8.0.1 -> 8.1.0

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/python-pillow/Pillow/blob/master/CHANGES.rst#810-2020-01-02

Fixes: CVE-2020-35654, CVE-2020-35653, CVE-2020-35655

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
That should go trough staging because the amounts of rebuilds. The diff won't apply cleanly because pillow got converted into a generic.nix file for pilleow-simd.

**Comment by mweinelt** [comment]
Hah, funny. I checked the last pillow update and that was in the 100-500 region. That was only a year ago.

https://github.com/NixOS/nixpkgs/pull/77714

File: pkgs/development/python-modules/pillow/default.nix [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
 
   disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "11c5c6e9b02c9dac08af04f093eb5a2f84857df70a7d4a6a6ad461aca803fb9e";
+    sha256 = "1fnvc07m15fv59bdlkds1q7rl9jrr3cbbn69sfqvzvdpjbknhxl8";
   };
 
   meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pillow/default.nix:17]: Unrelated but the license domain does not resolve anymore. I think https://github.com/python-pillow/Pillow/blob/master/LICENSE would fit.

Comment (mweinelt) [reply pkgs/development/python-modules/pillow/default.nix:17]: > Like PIL, Pillow is licensed under the open source HPND License:

So `licenses.hpnd`.

Comment (mweinelt) [reply pkgs/development/python-modules/pillow/default.nix:17]: Updated in https://github.com/NixOS/nixpkgs/pull/111655/commits/b5fa81e5c19cd9e8dc720bff95d34105aa77122f.

**Review by SuperSandro2000 — APPROVED** [review]
diff LGTM

**Pull request merged**